File: Java/Array/Alternates in an Array.java
//https://www.geeksforgeeks.org/problems/print-alternate-elements-of-an-array/1
class Solution {
    // function to print alternate elements of an array
    public ArrayList<Integer> getAlternates(int arr[]) {
        // Code Here
         ArrayList<Integer> ans = new ArrayList<>();
        for (int i = 0; i < arr.length; i += 2) {
            ans.add(arr[i]);
        }
        return ans;
    }
}

File: Cpp/Array/Alternates in an Array.cpp
//https://www.geeksforgeeks.org/problems/print-alternate-elements-of-an-array/1
class Solution {
  public:
    vector<int> getAlternates(vector<int> &arr) {
        // code here
        vector<int>ans;
        for(int i=0;i<arr.size();i=i+2)
        {
            ans.push_back(arr[i]);
        }
        return ans;
    }
};
